File: LAPORAN PRATIKUM/MODUL6/CODEBLOKS DAN PYTHON/SOAL 3/PRAK503-2210817210014-MuhammadQoharyRawidafany.py
def maksimal (a,b):
    return max(a,b)
def minimal (a,b):
    return min(a,b)

batas = 0
maks = -10000
minim = 100000
bilangan = int (input())
while batas < bilangan :
    nilai = map(int,input().split())
    for nilai_input in nilai :
        maks = maksimal(maks,nilai_input)
        minim = minimal(minim,nilai_input)
        batas +=1
print(maks,minim)

File: LAPORAN PRATIKUM/MODUL6/CODEBLOKS DAN PYTHON/SOAL 3/PRAK503-2210817210014-MuhammadQoharyRawidafany.c
#include <stdio.h>
int minimal (int a, int b){
    if (a<b){
        return a;
    }
    else if(b<a){
        return b;
    }
}
int maksimal (int a, int b){
    if (a>b){
        return a;
    }
    else if(b>a){
        return b;
    }
}
int main()
{
    int batas = 0;
    int maks = -100000;
    int minim = 100000;
    int bilangan;
    scanf("%d", &bilangan);
    while(batas < bilangan){
    int nilai;
    scanf("%d", &nilai);
    maks = maksimal(maks, nilai);
    minim = minimal(minim, nilai);
    batas++;
    }
    printf("%d %d",maks,minim);
}
